test/e_chronicler_web/controllers/journal_entry_controller_test.exs

```diff
@@ -55,4 +55,27 @@ defmodule EChroniclerWeb.JournalEntryControllerTest do
     assert html_response(conn, 400) =~ JournalEntryController.empty_entry_field_error
   end
 
+
+
+  test "renders form for editing chosen journal_entry", %{conn: conn} do
+    {:ok, journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(@valid_journal_entry)
+    conn = get(conn, Routes.journal_entry_path(conn, :edit, journal_entry))
+    assert html_response(conn, 200) =~ "Edit Journal Entry"
+  end
+
+  test "update redirects when data is valid", %{conn: conn} do
+    {:ok, journal_entry} = EChronicler.Models.JournalEntry.create_journal_entry(@valid_journal_entry)
+    conn = put(conn, Routes.journal_entry_path(conn, :update, journal_entry), journal_entry: @updated_journal_entry)
+    assert redirected_to(conn) == Routes.journal_entry_path(conn, :show, journal_entry)
+
+    conn = get(conn, Routes.journal_entry_path(conn, :show, journal_entry))
+    assert html_response(conn, 200) =~ "some updated author"
```

Comment:
This second part here maybe should be a separate test. The previous assert is testing what the test description says, i.e. that the redirect happens. In this assert, you're testing that the journal entry was updated successfully.